### PR TITLE
Centralization of some OpenAPI docs

### PR DESCRIPTION
**Pull request by ehuelsmann**
And at the same time add more documentation (instead of tripple dots)

### CODE REVIEW

File: lib/LedgerSMB/Routes/ERP/API/Languages.pm [hunk 1]
```diff
@@ -274,7 +274,7 @@ paths:
       operationId: getLanguages
       responses:
         200:
-          description: ...
+          description: Returns the full set of configured languages
```
Comment (ylavoie) [inline lib/LedgerSMB/Routes/ERP/API/Languages.pm:277]: openapi & info sections left purposely?

**Review by ylavoie — CHANGES_REQUESTED** [review]
openapi & info sections in Invoices should be checked too.

File: lib/LedgerSMB/Routes/ERP/API/Languages.pm (hunk 1 above)
Comment (ehuelsmann) [reply lib/LedgerSMB/Routes/ERP/API/Languages.pm:273]: No; accidentally left behind. Fix pushed. Thanks for the review!

**Comment by ehuelsmann** [comment]
> openapi & info sections in Invoices should be checked too.

@ylavoie this one is actually intentional: Invoices isn't using the DSL keywords from the router to declare API entrypoints (yet). Which makes it different from the others in that it uses its  own OpenAPI validator instead of the one from Router.pm. You're right, however, that this should be changed. I'll have a look at how difficult it is to change this.

**Comment by ehuelsmann** [comment]
@ylavoie i think I can fix https://github.com/ledgersmb/LedgerSMB/actions/runs/4913494760/jobs/8773795061?pr=7386#step:19:330 by overriding discount to be a float instead of a bigfloat. Or should we change the schema?

**Pull request merged**